hierarchicalforecast/utils.py

```diff
@@ -271,7 +285,16 @@ def aggregate(
                 unique_id = np.repeat(S_df.index, len(dates)),
                 ds = np.tile(dates, len(S_df.index)),
                 y = np.concatenate([y_agg, y_bottom], axis=0)))
-    Y_df = Y_df.set_index('unique_id').dropna()
+
+    Y_df = Y_df.set_index("unique_id").dropna()
+
+    if S_df.index.nunique() != Y_df.index.nunique():
+        print(
+            "Warning: Some time series were dropped from Y_df. "
+            "Are you sure your time series don't have missing entries?"
+        )
```

Comment:
I think we can leave this for a follow up PR where we try to directly address #211

Reply:
Yeah, I can remove it from this PR. Note however that it's a major annoyance for the users.

Reply:
Done